test/test_zipfile.py

```diff
@@ -0,0 +1,109 @@
+import os
+import sys
+import hashlib
+import subprocess
+import itertools
+import pytest
+try:
+    from tempfile import TemporaryDirectory
+except ImportError:
+    from backports.tempfile import TemporaryDirectory
+try:
+    from inspect import signature
+except ImportError:
+    from funcsigs import signature
```

Comment:
These fallbacks are probably unnecessary, given the Python 3.6+ requirement.

Reply:
done.

excuse: this file was 90% copy of https://github.com/cielavenir/zipfile39/blob/master/test/test_zipfile.py, where Python2 is supported.

Reply:
Oh, I think we should at least give credit to the original file in a comment.

Reply:
sure 

I have not thought of the credit in depth as it was written by me

maybe I should add the credit for further reference (for those who came here before visiting zipfile39 repo)